### ISSUE TITLE
i18N: localize with Ivy regression for equiv-text

### ISSUE THREAD

**Issue by cexbrayat** [angular/angular#38694]
# 🐞 bug report

### Affected Package
The issue is caused by package @angular/localize in v10.1.0

### Is this a regression?

Yes, the previous version in which this bug was not present was: v10.0.0

### Description

Now that the CLI allows to extract the messages with Ivy, I noticed a difference in the extracted messages.
The extracted text is surrounded by comments, but the extracted messages with VE were fine.
With Ivy, it changes to:

```diff
-        <source>Welcome to Ponyracer <x id="INTERPOLATION" equiv-text="{{ user.firstName }}"/> <x id="INTERPOLATION_1" equiv-text="{{ user.lastName }}"/>!</source>
+       <source>Welcome to Ponyracer <x id="INTERPOLATION" equiv-text="ributes[] --&gt;
+ &lt;"/> <x id="INTERPOLATION_1" equiv-text="y.gif&quot;
+   alt=&quot;r"/>!</source>
```

## 🔬 Minimal Reproduction

See https://github.com/cexbrayat/localize-ivy-repro

`app.component.html` has a bunch of i18n introduced in the second commit.
Run `ng xi18n --output-path src/locale/` to extract the messages with VE (result in 3rd commit)
Run `ng xi18n --ivy --output-path src/locale/` to extract the messages with Ivy (result in 4th commit)
Most differences are expected, except the regression indicated above.

## 🌍  Your Environment

**Angular Version:**
<pre><code>

Angular CLI: 10.1.0
Node: 14.6.0
OS: darwin x64

Angular: 10.1.0
... animations, cli, common, compiler, compiler-cli, core, forms
... localize, platform-browser, platform-browser-dynamic, router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.1001.0
@angular-devkit/build-angular     0.1001.0
@angular-devkit/build-optimizer   0.1001.0
@angular-devkit/build-webpack     0.1001.0
@angular-devkit/core              10.1.0
@angular-devkit/schematics        10.1.0
@ngtools/webpack                  10.1.0
@schematics/angular               10.1.0
@schematics/update                0.1001.0
rxjs                              6.6.2
typescript                        4.0.2
webpack                           4.44.1
</code></pre>

**Comment by petebacondarwin** [comment]
I think this should be fixed by https://github.com/angular/angular/pull/38645

**Comment by AndrewKushnir** [comment]
FYI, the corresponding PR #38645 was merged and the fix will be included into 10.1.1 release. Thank you.
